gradle/publishing.gradle

```diff
@@ -13,9 +13,14 @@ java {
 publishing {
   repositories {
     maven {
+      name = 'sonatype'
       def releasesRepoUrl = "https://oss.sonatype.org/service/local/staging/deploy/maven2/"
       def snapshotsRepoUrl = "https://oss.sonatype.org/content/repositories/snapshots/"
       url = isReleaseVersion ? releasesRepoUrl : snapshotsRepoUrl
+      credentials {
+        username nexusUsername
```

Comment:
Leads to an exception when I'm trying to build the branch locally:
```
FAILURE: Build failed with an exception.

* Where:
Script '.../workspace/config-view/gradle/publishing.gradle' line: 21

* What went wrong:
A problem occurred evaluating script.
> Could not get unknown property 'nexusUsername' for Credentials [username: null] of type org.gradle.internal.credentials.DefaultPasswordCredentials_Decorated.
```
I am not sure where you keep your credentials properties, but there is [a nice workaround](https://discuss.gradle.org/t/best-practices-for-configuring-maven-publish-credentials-and-how-do-i-skip-configuring-publications-unless-actually-publishing/13173/2) which enables build even without actually publishing.

Reply:
fixed, I didn't realize this wouldn't work without defaults